src/chrome/content/rules/welmers.net.xml

```diff
@@ -0,0 +1,18 @@
+<ruleset name="welmers.net">
+
+	<!--	Direct rewrites:
+				-->
+	<target host="welmers.net" />
+	<target host="www.welmers.net" />
+	<target host="www4.welmers.net" />
+	<target host="www6.welmers.net" />
+	<target host="users.welmers.net" />
+	<target host="gallery.welmers.net" />
+
+
+	<securecookie host="^(www.?|users|gallery)\.welmers\.net$" name=".+" />
```

Comment:
Can you send the cookies with `Secure` flag on the server side? If HTTPS Everywhere rewrites a cookie to include `Secure` flag and then user uninstalls HTTPS Everywhere the flag will not be removed and it won't be available on insecure version of website.